### ISSUE TITLE
Version Doesn't Work for Docker

### ISSUE THREAD

**Issue by mielliott** [bio-guoda/preston#11]
When running
```shell
# docker run -v ~/tmp/:/data bio.guoda/preston:0.0.16 version
dev
```
Expected
```shell
# docker run -v ~/tmp/:/data bio.guoda/preston:0.0.16 version
0.0.16
```

**Comment by mielliott** [comment]
For configuration on how preston images are built - https://github.com/bio-guoda/preston/blob/master/pom.xml#L140 / notice the "update" default argument

**Comment by mielliott** [comment]
The version was being stamped into MANIFEST.MF by the maven-assembly-plugin in pom.xml. However, it seems that jib (which builds the docker image) does not include the manifest file ([or anything else included after maven's compile phase](https://github.com/GoogleContainerTools/jib/issues/1470#issuecomment-462423696)).

Instead, I added a resouces/preston.properties file containing ```version=${project.version}```, configured pom.xml to substitute the correct version number, then changed Preston.java's ```getVersion()``` function to output the version listed in preston.properties. See 9218cf5f9960c619ab5027d87a61e0527ae2ea0d and https://rterp.wordpress.com/2012/03/16/stamping-version-number-and-build-time-in-properties-file-with-maven/.

I have not removed the version stamping from MANIFEST.MF. Should we just remove the version manifest entries, or can we remove the manifest altogether since it won't be included in the docker image?
https://github.com/bio-guoda/preston/blob/9218cf5f9960c619ab5027d87a61e0527ae2ea0d/pom.xml#L124-L130

**Comment by jhpoelen** [comment]
Looks great! Suggest to leave manifest version stamping in for now. Doesn't hurt. Also, suggest to close this issue after making release 0.1.1 and verifying that the version in fact shows up using docker image.

**Comment by mielliott** [comment]
Verified that it's fixed by 86a27c53fd7d91ccdbd1d4f04121572afd2deb97 in release 0.1.1.

**Comment by jhpoelen** [comment]
Nice!
